locations/spiders/alpha_pharm_za.py

```diff
@@ -0,0 +1,43 @@
+import ast
+
+from locations.categories import Extras, apply_yes_no
+from locations.hours import OpeningHours
+from locations.json_blob_spider import JSONBlobSpider
+
+
+class AlphaPharmZASpider(JSONBlobSpider):
+    name = "alpha_pharm_za"
+    item_attributes = {
+        "brand": "Alpha Pharm",
+        "brand_wikidata": "Q116487265",
+    }
+    allowed_domains = ["alphapharmacies.co.za"]
```

Comment:
Minor comment: why bother with `allowed_domains` as this is not a `CrawlSpider` that may spiral out of control.

Reply:
Thanks for the tip. I copied from elsewhere and didn't think about it I guess. I've now removed that.